### PR TITLE
New UI

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@akatsoulas or @johngian r?

**Comment by akatsoulas** [comment]
screen shot 2016-11-24 at 12 14 50
 The footer seems broken. This is from the pre-selected list of devices, emulating an iphone 6s

**Comment by akatsoulas** [comment]
screen shot 2016-11-24 at 12 20 53

I would prefer the text centered here.

File: moderator/moderate/forms.py [hunk 1]
```diff
@@ -12,9 +12,14 @@ class QuestionForm(forms.ModelForm):
     """Question Form."""
     question = forms.CharField(
         validators=[MaxLengthValidator(140), MinLengthValidator(10)],
-        max_length=140, widget=forms.TextInput(attrs={'placeholder': QUESTION}))
+        max_length=140, widget=forms.TextInput(attrs={'placeholder': QUESTION,
+                                                      'class': 'form-control',
```
Comment (akatsoulas) [inline moderator/moderate/forms.py]: Why are you passing the class here?

**Comment by comzeradd** [comment]
> Shouldn't the text be centered here? Moreover, I would prefer the login link next to tabzilla as it was before. Thoughts?

I actually prefer the Login to be centered bellow the text, since it's the only action you have to do when not logged in. I pushed the change along with a minor change for the footer issue.

**Comment by akatsoulas** [comment]
screen shot 2016-11-24 at 17 14 05
The error message is broken here.

**Comment by akatsoulas** [comment]
Also the text in the footer feels wrong.  Maybe we could move the `mozilla` log below the text

**Comment by comzeradd** [comment]
Good catch. The hamburger is no longer needed when logged out. I fixed it, along with login error message.

What do you mean about the footer? It's similar to what other sites use. (eg. [toolkit](https://toolkit.mozilla.org/), [science](https://science.mozilla.org/))

**Comment by akatsoulas** [comment]
The text with the mozilla logo, under the three icons (contribute, legal, license) looks a big funny the way it is right now (last screenshot). Maybe we could center the text under the three icons and move the mozilla logo under the text. We can at least try it to see if it looks better that way. Also with this proposal the footer will be similar to the way the text in body is aligned.

**Comment by akatsoulas** [comment]
Or we could have it the other way around. Logo under the icons and under that the text.

**Comment by comzeradd** [comment]
Here are two screenshots. I prefer the first one, especially now that I added one more icon on top.

screen shot 2016-11-26 at 16 29 21-fullpage

screen shot 2016-11-26 at 16 35 28-fullpage

**Comment by akatsoulas** [comment]
event_archive_mobile
The UI in the `event archive` is different from the rest of the project.

**Comment by akatsoulas** [comment]
mobile_footer
Let's make the footer similar to the science site for the mobile view.

**Comment by akatsoulas** [comment]
error_msg_left
The error message should be centered here.

**Comment by akatsoulas** [comment]
mobile_view_icon
The alignment of the icon is a bit off here.

**Comment by akatsoulas** [comment]
question_error_msg
The error message is not aligned correctly in the mobile view.

File: moderator/moderate/templates/base.jinja [hunk 2]
```diff
@@ -7,94 +7,114 @@
...
             {% if user.is_authenticated() %}
-              <nav id="nav-main" role="navigation">
-                <a href="{{ url('archive') }}">Event archive</a>
-                <a href="{{ url('oidc_logout') }}">Logout</a>
-              </nav>
-            {% else %}
-              <a class="login button" href="{{ url('oidc_authentication_init') }}">
-                Login
-              </a>
+              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
```
Comment (akatsoulas) [inline moderator/moderate/templates/base.jinja]: Let's keep the line length to 100 chars.

File: moderator/moderate/templates/base.jinja [hunk 3]
```diff
@@ -7,94 +7,114 @@
...
+
+      <div class="container">
+
+        {% block breadcrumb %}
+        {% endblock %}
+
+        {% if messages %}
+          {% for message in messages %}
+            <div class="alert alert-{{ message.tags }} alert-dismissible" role="alert">
+              <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```
Comment (akatsoulas) [inline moderator/moderate/templates/base.jinja]: Let's keep the line length to 100 chars.

File: moderator/moderate/templates/index.jinja [hunk 4]
```diff
@@ -1,29 +1,32 @@
...
   {% endfor %}
+
 {% else %}
-  <hgroup id="main-feature" class="center">
-    <h2>Let your voice be heard.</h2>
-    <p>Mozilla Moderator is a panel moderation system for submitting and voting on questions.</p>
-    <p>Please log in using your vouched mozillians.org account.</p>
-  </hgroup>
+  <h3 class="text-center">Let your voice be heard.</h3>
+  <p class="text-center">Mozilla Moderator is a panel moderation system for submitting and voting on questions.</p>
```
Comment (akatsoulas) [inline moderator/moderate/templates/index.jinja]: Length please :)

File: moderator/moderate/templates/questions.jinja [hunk 5]
```diff
@@ -1,109 +1,121 @@
...
-  <small class="error">Please correct the errors below</small>
-{% endif %}
+  <ol class="breadcrumb">
+    <li><a href="/">Events</a></li>
+    <li class="active">{{ event.name }}</li>
+  </ol>
+
+  {% if q_form.errors %}
+    <div class="alert alert-danger alert-dismissible" role="alert">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
```
Comment (akatsoulas) [inline moderator/moderate/templates/questions.jinja]: Same here.

Comment (akatsoulas) [inline moderator/moderate/templates/questions.jinja]: This has multiple occurrences throughout the project.

**Comment by comzeradd** [comment]
Thanks for the thorough review. I fixed the issues mentioned, adjust the mobile view footer and did a full pass on the archive page.

**Pull request merged**